examples/basics/scene/volume.py

```diff
@@ -64,6 +64,12 @@
 cam3 = scene.cameras.ArcballCamera(parent=view.scene, fov=fov, name='Arcball')
 view.camera = cam2  # Select turntable at first
 
+# Create an XYZAxis visual
+axis = scene.visuals.XYZAxis(parent=view)
+s = STTransform(translate=(50, 50), scale=(50, 50, 50, 1))
+affine = s.as_matrix()
+print affine
+axis.transform = affine
```

Comment:
have you tried doing `axis.transform = s` instead? Hopefully you don't have to do the `.as_matrix()` to get it to work...?